apps/acrs/translations/src/en/fields.json

```diff
@@ -200,14 +200,15 @@
       }
     }
   },
-  "family-in-uk": {
-    "label": "SKELETON: family-in-uk",
+  "has-family-in-uk": {
+    "legend": "Do you have family that live in the UK?",
+    "label": "Do you have family that live in the UK?",
```

Comment:
I think if you use legend in this case and remove label then in acrs/fields/index.js for this one you can use `legend: { className: 'bold' }` instead of visuallyHidden

Reply:
Yes i made a mistake in there i was trying both out forgot to delete the label